src/HealthGPS/kevin_hall_model.cpp

```diff
@@ -265,15 +330,54 @@ double KevinHallModel::compute_AT(double EI, double EI_0) const {
     return beta_AT * delta_EI;
 }
 
+void KevinHallModel::initialise_weight(Person &person) {
```

Comment:
Ok, this initialisation is more complicated that what I included. Did Ali change the implementation along the way?